README.md

```diff
@@ -258,18 +258,12 @@ a Python version greater than 2.7 or 3.4 (Python <=3.3 is no longer supported).
 * **SciPy**: Import Matlab workspace files in the Variable Explorer.
 * **Cython**: Run Cython files in the IPython console.
 
-
-## Backers
-
-Support us with a monthly donation and help us continue our activities.
-
-[![Backers](https://opencollective.com/spyder/backers.svg)](https://opencollective.com/spyder#support)
-
-
 ## Sponsors
 
 Become a sponsor to get your logo on our README on Github.
 
+[![Quansight](https://static.wixstatic.com/media/095d2c_2508c560e87d436ea00357abc404cf1d~mv2.png/v1/crop/x_0,y_9,w_915,h_329/fill/w_191,h_66,al_c,usm_0.66_1.00_0.01/095d2c_2508c560e87d436ea00357abc404cf1d~mv2.png)](https://www.quansight.com/) [![Numfocus](https://numfocus.org/wp-content/uploads/2018/01/optNumFocus_LRG.png)](https://numfocus.org/)
```

Comment:
Please make Quansight and Numfocus bigger than the Open Collective image below (about the same size you use in the webpage).

Reply:
![imagen](https://user-images.githubusercontent.com/16781833/52223743-4de9dc00-2874-11e9-9ba5-7f0d049e8013.png)

Reply:
Please make the Quansight and Numfocs logos even bigger (they have to look bigger than the Open collective image) and add the rest of the text we added in the webpage:

> and the donations we have received ...

----

So the idea is to change this section to be as close as possible as the Sponsors section you added to our webpage.

Reply:
![imagen](https://user-images.githubusercontent.com/16781833/52226107-0cf4c600-287a-11e9-96bd-dee50948837a.png)